.github/workflows/rust.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-  GST_PLUGIN_FEATURE_RANK: pulsesink:NONE,alsasink:NONE,jacksink:NONE,fakesink:MAX
+  GST_PLUGIN_FEATURE_RANK: openalsink:NONE,pulsesink:NONE,alsasink:NONE,jacksink:NONE,fakesink:MAX
```

Comment:
I am not sure if i understand this change. Is it relevant to this media issue?

Reply:
Sorry, I'm trying to debug the CI failure which is unrelated to this change.